src/utils/TreeUtils.js

```diff
@@ -1,4 +1,9 @@
-function visitTree(elt, observeCb) {
+/**
+ * Recursively visit nodes if the node has children
+ * @param {Object} elt tree element
+ * @param {function} observeCb obeserve function
```

Comment:
@param {function} observeCb Callback for each child/element pair: observeCb(child, elt).